composer.json

```diff
@@ -23,7 +23,14 @@
         "doctrine/orm": "^2.5",
         "symfony/options-resolver": "^3.0|^4.0",
         "symfony/property-access": "^3.0|^4.0",
-        "friendsofsymfony/jsrouting-bundle": "^1.6|^2.0"
+        "symfony/translation": "^3.0|^4.0",
+        "symfony/security": "^3.0|^4.0",
+        "sensio/generator-bundle": "^3.1",
+        "friendsofsymfony/jsrouting-bundle": "^1.6|^2.0",
+        "twig/twig": "^2.4"
```

Comment:
Why did you add all of this dependencies?

Reply:
This are the dependencies who are missing and on which this library depends.

Reply:
I will remove this, because it's no part of this PR (issue) and should be fixed in a separate issue.